Competition2017/src/competition/subsystems/shooter_belt/commands/RunBeltTracerPowerMode.java

```diff
@@ -0,0 +1,26 @@
+package competition.subsystems.shooter_belt.commands;
+
+import competition.subsystems.hybrid.commands.BaseShooterBeltAndWheelCommand;
+import competition.subsystems.shooter_belt.ShooterBeltSubsystem;
+import competition.subsystems.shooter_wheel.ShooterWheelSubsystem;
+
+public class RunBeltTracerPowerMode extends BaseShooterBeltAndWheelCommand {
```

Comment:
I think there might be a problem here, this base class requires the shooterWheel which means the wheel wouldn't actually spin to shoot because this command only actuates the belts?